flocker/control/_protocol.py

```diff
@@ -123,46 +124,27 @@ def fromBox(self, name, strings, objects, proto):
         self.another_argument.fromBox(name, strings, objects, proto)
 
 
-class CachingEncoder(object):
-    """
-    Cache results of ``wire_encode`` and re-use them, relying on the fact
-    we're encoding immutable objects.
+# The configuration and state can get pretty big, so don't want too many:
+_wire_encode_cache = LRUCache(50)
```

Comment:
What's defined as pretty big and too many? I guess we can watch this value over time and modify the cache size if necessary.